.github/workflows/test.yml

```
@@ -0,0 +1,33 @@
+name: Run unit and ui tests
+
+on:
+  push:
+  
+jobs:
+  test:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Setup Ruby 3.2.1
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.2.1'
+          bundler-cache: true
+      
+      - name: Netrc api.github.com
```

```suggestion
      - name: Netrc maven.pkg.github.com
```
Seems as I forgot to update this in the publish workflow

---

What about that? 😅

---

this is already fixed 🤔

---

No, the command is correct, but the name of the lane is outdated 😅